sample/SampleHost/Program.cs

```diff
@@ -4,7 +4,6 @@
 using System;
 using Microsoft.Azure.WebJobs;
 using Microsoft.Azure.WebJobs.Logging;
-using Microsoft.Azure.WebJobs.ServiceBus;
```

Comment:
The new EH sample function demonstrates the problem nicely. Today I added the EH sample function to this app, but after referencing the extension assembly, doing "config." for intellisense didn't show "UseEventHub".